docs/reference/ilm/set-up-lifecycle-policy.asciidoc

```diff
@@ -57,18 +65,26 @@ PUT _ilm/policy/my_policy
 
 <1> Roll over the index when it reaches 25GB in size
 <2> Delete the index 30 days after rollover
+====
 
 [discrete]
 [[apply-policy-template]]
 === Apply lifecycle policy with an index template
 
 To use a policy that triggers the rollover action, 
 you need to configure the policy in the index template used to create each new index.
- 
-In addition to specifying the name of the policy in the `index.lifecycle.name` setting, 
-you specify a `index.lifecycle.rollover_alias` for referencing
-the indices managed by this policy. 
+You specify the name of the policy and the alias used to reference the rolling indices.
+
+To use the Create template wizard to create a template from {kib} Management,
+go to **Management > Elasticsearch > Index Management** and select the Index Templates view.
+
+image:images/ilm/create-template-wizard.png[]
+
+The <<indices-put-template,put template API>> is invoked to add templates to a cluster.
```

Comment:
Rewords passive voice. Previously, it wasn't clear if this was an extra step the user needed to take or if we're just showing what Kibana was doing under the hood.
```suggestion
The wizard invokes the <<indices-put-template,put template API>> to add templates to a cluster.
```